packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -22,6 +22,7 @@ const testContext = vm.createContext({
   transform: babel.transform,
   setTimeout: setTimeout,
   setImmediate: setImmediate,
+  expect,
```

Comment:
Can you also use `expect` in `./helpers`?

Reply:
I had left it to be migrated when we are in a stage to completely migrate away from `assert`. Anyway, at most what I can change *directly* is `assertNoOwnProperties`. We can't migrate the other helper `assertArrayEquals` without running the code-mod at related places (I plan to do it a separate PR).